circle-mlir/circle-mlir/tools-test/onnx2circle-value-test/util_validation.py

```diff
@@ -19,7 +19,10 @@ def get_peir_from(baseline_output, target_output):
         PEIR = 0
     else:
         INTERVAL = baseline_max - baseline_min
-        PEIR = PEAK_ERROR / INTERVAL
+        if INTERVAL == 0:
+            PEIR = 0
```

Comment:
I don't know much about PEIR but if INTERVAL is 0, `PEAK_ERROR / INTERVAL` should go to `infinite`... but you suggested to 0. can you share your thoughts about this?

Reply:
Mathematically, if `INTERVAL` is `0`, the result of `PEAK_ERROR / INTERVAL` should be `infinite`. While this makes sense mathematically, I'm not entirely sure if applying this logic is appropriate for `PEIR` in our context. However, it does seem more reasonable than setting it to `0`, so I’ll apply your suggestion.